Dockerfile

```diff
@@ -94,6 +94,11 @@ RUN --mount=type=cache,target=/var/lib/apt/lists \
 RUN --mount=type=cache,target=/root/.cache \
     --mount=type=bind,from=poetry,source=/tmp,target=/poetry \
     python3 -m pip install --disable-pip-version-check --no-deps --requirement=/poetry/requirements-dev.txt
+ENV PATH=/root/.local/bin/:${PATH}
+
+WORKDIR /opt/c2cgeoportal
+COPY ci/applications*.yaml .
+RUN c2cciutils-download-applications --applications-file=applications.yaml --versions-file=applications-versions.yaml
```

Comment:
Here you install an application from `ci/applications.yaml` using `c2cciutils` in `~/.local/bin` (probably in user root) in a Docker image that may be runned on development, integration and production environments.

I was thinking c2cciutils-download-application was only for usage on CI host.

Hope this application will not be used at run time.

Reply:
Here we are in the tools stage => tools image, then it will not be in the running containers :-)